.travis.yml

```diff
@@ -1,8 +1,13 @@
-language: php
-php:
- - hhvm-3.30
- 
+sudo: required
+language: generic
+services: docker
+env:
+- HHVM_VERSION=3.30.12
```

Comment:
If there is a 3.30-latest tag, you can use that instead. The 3.29 branch used an exact version because no latest was available. But then again, hhvm 3.30.13 won't ever be released either...

https://hub.docker.com/layers/hhvm/hhvm/3.30-lts-latest/images/sha256-40e3a585d350bfdac45180364a4d352782fb7cbca05a7e0aa56d964262c9551e?context=explore

Reply:
It's `3.30-lts-latest` (we've dropped the `-lts` part from more recent builds for consistency)

I've just pushed a `3.29-latest` tag